### PR TITLE
coq-elpi 1.3.0

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: released/packages/coq-elpi/coq-elpi.1.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+set of Coq's primitives, e.g. printing a message, accessing the
+environment of theorems and data types, defining a new constant and so on.
+For convenience it also provides a quotation and anti-quotation for Coq's
+syntax in λProlog.  E.g. `{{nat}}` is expanded to the type name of natural
+numbers, or `{{A -> B}}` to the representation of a product by unfolding
+the `->` notation. Finally it provides a way to define new vernacular commands
+and new tactics."""
+url {
+src: "https://github.com/LPCIC/coq-elpi/archive/v1.3.0.tar.gz"
+checksum: "sha256=d04f2e3a53e2ec1b872b7ab067b190c615d03ce6f4cddfb5c73a5594c4cd0e35"
```
Comment (palmskog) [inline released/packages/coq-elpi/coq-elpi.1.3.0/opam]:
```suggestion
checksum: "sha256=d93545398683231159c1466456ba9d9920fb3ceaecf80c49827be11816762581"
```
Looks like it all gets stuck on the checksum, this is what I got after download.

File: released/packages/coq-elpi/coq-elpi.1.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+name: "coq-elpi"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" ]
+license: "LGPL 2.1"
```
Comment (palmskog) [inline released/packages/coq-elpi/coq-elpi.1.3.0/opam]:
```suggestion
license: "LGPL-2.1-or-later"
```

File: released/packages/coq-elpi/coq-elpi.1.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+name: "coq-elpi"
```
Comment (palmskog) [inline released/packages/coq-elpi/coq-elpi.1.3.0/opam]:
```suggestion
```

**Comment by gares** [comment]
many thanks

**Pull request merged**